### DISCUSSION TITLE
How to have Tabby automatically be in my WSL2 Ubuntu home directory

### DISCUSSION THREAD

**Discussion by gesslar** [Eugeny/tabby#9871] [Q&A] [answered]
I am trying out Tabby as an alternative to Windows Terminal. One thing that is small to other people, but is somewhat important to me. I cannot find a setting in Tabby that upon opening a session will be in my WSL2 Ubuntu's home directory. It seems to always be in my Windows home directory and I have to remember to `cd` to get to the right spot.

**Comment by Kisaragi-ng** [comment, marked as answer]
have you tried to setup Working Directory in the Profile?

![image](https://github.com/user-attachments/assets/ec01602d-3b66-46c8-9837-acad937e8be0)

**Reply by gesslar** [reply]
How do you get there? When I go to Profiles & connections, and i select the triple dots to the built-in WSL / Ubuntu-22.04 profile, I only get Duplicate and Hide.

**Reply by gesslar** [reply]
Okay, I guess this required me making a custom profile, which I did based on the Ubuntu one I always use. And I set the Working directory to `~` but that didn't do anything. 😭

**Reply by gesslar** [reply]
Got it. You're a saviour, thank you!

`//wsl$/Ubuntu-22.04/home/gesslar/`

**Reply by MaxAlekseyev** [reply]
> ``

Can you send me screenshot?

**Comment by dendrown** [comment]
I needed backslashes, rather than slashes:
![image](https://github.com/user-attachments/assets/4d5ee22d-aceb-43c7-9ab2-fbabea38bcb1)

**Reply by MaxAlekseyev** [reply]
Working, thanks

**Comment by rluiten** [comment]
It should be possible to add "--cd ~" to the command line of wsl to have it work.
But this seems to not work in tabby.

I Tried to set the Working directory as @dendrown above shows to the path including 
`\\wsl.localhost\deb2\home\myname` as per his example this also does not work.

I know the `--cd ~` works as i use this in windows terminal.

Is the `--cd ~` being stripped by tabby maybe?
